website/pages/en/glossary.mdx

```diff
@@ -76,10 +76,10 @@ title: Glossary
 
 - **Cooldown Period**: The time remaining until an Indexer who changed their delegation parameters can do so again.
 
-- **L2 Transfer Tools**: Smart contracts and UI that enable network participants to transfer from Ethereum mainnet to Arbitrum One. Network participants can transfer delegated GRT, subgraphs, curation shares, and Indexer's self stake.
+- **L2 Transfer Tools**: Smart contracts and UI that enables network participants to transfer network related assets from Ethereum mainnet to Arbitrum One. Network participants can transfer delegated GRT, subgraphs, curation shares, and Indexer's self stake.
```

Comment:
Actually, I think this one was correct? It is the smart contracts _and_ the UI that _enable_ network participants to do that, right?
```suggestion
- **L2 Transfer Tools**: Smart contracts and UI that enable network participants to transfer network related assets from Ethereum mainnet to Arbitrum One. Network participants can transfer delegated GRT, subgraphs, curation shares, and Indexer's self stake.
```